modules/sstp/scanner.go

```diff
@@ -0,0 +1,228 @@
+// Package sstp contains the module implementation for SSTP HTTP(S) connection establishment
+//
+// The module performs a complete TLS handshake to then send an HTTP SSTP_DUPLEX_POST request. The default port is 443.
+package sstp
+
+import (
+	"crypto/tls"
+	"io"
+	"strings"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/zmap/zgrab2"
+)
+
+type TLSFlags struct {
+	zgrab2.BaseFlags
+	zgrab2.TLSFlags
+	File string `long:"file" default:"" description:"file to write responses to" `
+}
+
+type TLSScanner struct {
+	config *TLSFlags
+}
+
+type TLSModule struct {
+}
+
+// ScanResults instances are returned by the module's Scan function.
+type Results struct {
+	Status         string `json:"status,omitempty"`
+	Content_Length string `json:"content_length,omitempty"`
+	Host           string `json:"host,omitempty"`
+	Content_Type   string `json:"content_type,omitempty"`
```

Comment:
To analyze SSTP response, we actually need only the status, content length and host, so it would be possible to remove the rest. This would make the output much cleaner, however, our raw data also includes those headers. The modules would differ to when we used them for our analysis then.